README.md

```diff
@@ -1,60 +1,83 @@
+
+This journey is aimed at helping Spark developers quickly setup and run the TPC-DS benchmark in their own development setup. 
+
+When the reader has completed this journey, they will understand the following:
 
 * How to setup the TPC-DS toolkit 
 * How to generate TPC-DS datasets at different scale factor 
 * How to create Spark database artifacts 
 * How to run TPC-DS benchmark queries on Spark in local mode and see the results 
 * Things to consider when increasing the data scale and run against a spark cluster 
 
+![Architecture diagram](doc/source/images/architecture.png)
+
+## Flow
+1.
+2.
+3.
+
+## Included components
+
+* [Apache Spark](http://spark.apache.org/): An open-source, fast and general-purpose cluster computing system
+* [Jupyter Notebooks](http://jupyter.org/): An open-source web application that allows you to create and share documents that contain live code, equations, visualizations and explanatory text.
+
+## Featured technologies
+
+* [Data Science](https://medium.com/ibm-data-science-experience/): Systems and scientific methods to analyze
+structured and unstructured data in order to extract knowledge and insights.
+* [Artificial Intelligence](https://medium.com/ibm-data-science-experience): Artificial intelligence can be
+applied to disparate solution spaces to deliver disruptive technologies.
+* [Python](https://www.python.org/): Python is a programming language that lets you work more quickly
+and integrate your systems more effectively.
+
+# Steps
+
 There are two modes of excercising this journey:
-* By using a simple interactive command line shell script
-* By using a Jupyter notebook run in IBM Data Science Experience (DSX) on IBM Bluemix
+* Run locally using a simple interactive command line shell script.
+* Use a Jupyter notebook running in the IBM Data Science Experience (DSX) on IBM Bluemix. To run in this mode, jump down to the "Run using a Jupyter notebook ..." instructions listed below.
 
+## Run locally
+1. [Clone the repo](#1-clone-the-repo)
+2. [Setup developement tools](#2-setup-development-tools)
+3. [Install Spark](#3-install-spark)
+4. [Run the script](#4-run-the-script)
 
-# Quick Start
+### 1. Clone the repo
 
-## Running from command line.
+Clone the `spark-tpc-ds-performance-test` repo locally. In a terminal, run:
 
-### Setup
-#### Development Tools
-Make sure the required development tools are installed in your platform. This journey is supported on Mac and Linux
-platforms only.
+``` 
+git clone https://github.com/spark-tpc-ds-performance-test 
+```
+
+### 2. Setup development tools
+
+Make sure the required development tools are installed in your platform. This journey is supported on Mac and Linux platforms only.
```

Comment:
tack on a bit at the end: "This journey is supported on Mac and Linux platforms only, depending on your platform, run the following command to install the necessary development tools"